tests/test_stats.py

```
@@ -0,0 +1,613 @@
+    rng = np.random.default_rng(9)
+    idx = _daily_index(40)
+    frame = pd.DataFrame(
+        {
+            "a": rng.normal(scale=0.01, size=40),
+            "b": rng.normal(scale=0.02, size=40),
+        },
+        index=idx,
+    )
+    rs = ReturnSeries.from_returns(frame)
+    fitted = min_variance(rs)
+    train, test = chronological_split(rs, test_size=0.25)
```

Ruff (0.16.2)

[warning] Unpacked variable `train` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)